docs/source/api/deliveryservice_server_capabilities.rst

```diff
@@ -0,0 +1,222 @@
+	Access-Control-Allow-Headers: Origin, X-Requested-With, Content-Type, Accept, Set-Cookie, Cookie
+	Access-Control-Allow-Methods: POST,GET,OPTIONS,DELETE
+	Access-Control-Allow-Origin: *
+	Content-Type: application/json
+	Set-Cookie: mojolicious=...; Path=/; HttpOnly
+	Whole-Content-Sha512: UFO3/jcBFmFZM7CsrsIwTfPc5v8gUiXqJm6BNp1boPb4EQBnWNXZh/DbBwhMAOJoeqDImoDlrLnrVjQGO4AooA==
+	X-Server-Name: traffic_ops_golang/
+	Date: Mon, 07 Oct 2019 22:15:11 GMT
+	Content-Length: 396
+
+	{
+		"response": [
+			{
+				"deliveryServiceID": 1,
+				"lastUpdated": "2019-10-07 22:05:31+00",
+				"serverCapability": "ram"
+				"xmlId": "example_ds-1",
+			},
+			{
+				"deliveryServiceID": 2,
+				"lastUpdated": "2019-10-07 22:05:31+00",
+				"serverCapability": "disk"
+				"xmlId": "example_ds-2,
+			}
+		]
+	}
+
+``POST``
+========
+Associates a Server Capability to a Delivery Service.
+
+:Auth. Required: Yes
+:Roles Required: "admin" or "operations"
+:Response Type:  Object
+
+Request Structure
+-----------------
+:deliveryServiceID: The Delivery Service's ID to associate
+:serverCapability:  The Server Capability's name to associate
+
+.. code-block:: http
+	:caption: Request Example
+
+	POST /api/1.4/deliveryservice_server_capabilities HTTP/1.1
+	Host: trafficops.infra.ciab.test
+	User-Agent: curl/7.47.0
+	Accept: */*
+	Cookie: mojolicious=...
+	Content-Length: 56
+	Content-Type: application/json
+
+	{
+		"deliveryServiceID": 1,
+		"serverCapability": "disk"
+	}
+
+Response Structure
+------------------
+:deliveryServiceID: The Delivery Service's ID
+:lastUpdated:       The date and time at which this association between the Delivery Service and the Server Capability was last updated, in an ISO-like format
+:serverCapability:  The Server Capability's name
+
+.. code-block:: http
+	:caption: Response Example
+
+	HTTP/1.1 200 OK
+	Access-Control-Allow-Credentials: true
+	Access-Control-Allow-Headers: Origin, X-Requested-With, Content-Type, Accept, Set-Cookie, Cookie
+	Access-Control-Allow-Methods: POST,GET,OPTIONS,DELETE
+	Access-Control-Allow-Origin: *
+	Content-Type: application/json
+	Set-Cookie: mojolicious=...; Path=/; HttpOnly
+	Whole-Content-Sha512: eQrl48zWids0kDpfCYmmtYMpegjnFxfOVvlBYxxLSfp7P7p6oWX4uiC+/Cfh2X9i3G+MQ36eH95gukJqOBOGbQ==
+	X-Server-Name: traffic_ops_golang/
+	Date: Mon, 07 Oct 2019 22:15:11 GMT
+	Content-Length: 287
+
+	{
+		"alerts": [
+			{
+				"level": "success"
+				"text": "deliveryservice server_capability was updated.",
+			}
+		],
+		"response": {
+			"deliveryServiceID": 1,
+			"lastUpdated": "2019-10-07 22:15:11+00",
+			"serverCapability": "disk"
+		}
+	}
+
+``DELETE``
+==========
+De-associate a Server Capability from a Delivery Service.
+
+:Auth. Required: Yes
+:Roles Required: "admin" or "operations"
+:Response Type:  ``undefined``
+
+Request Structure
+-----------------
+:deliveryServiceID: The Delivery Service's ID to de-associate
+:serverCapability:  The Server Capability's name to de-associate
```

Comment:
nit: should probably be "dissociate"

Reply:
I agree that is better. I found another use of "de-associate" in the `cachegroupparameters/{{ID}}/{{parameterID}}` docs. Should I update that entry in the PR as well?

Reply:
yeah, if you feel like it. It's not necessary, but that'd be nice.